RELEASING.md

```diff
@@ -0,0 +1,12 @@
+Release procedure for dependency-analysis-android-gradle-plugin
+
+1. Update the changelog
+1. Bump version number to next stable version (use semantic versioning: x.y.z)
+1. Update README if needed
+1. git commit -am "Prepare for release x.y.z." 
+1. Release on ...
+1. git tag -a x.y.z -m "Version x.y.z"
```

Comment:
Small question. Are you saying tags should always be `x.y.z`? I think I've been using `vx.y.z`.